fsthttp/backend.go

```diff
@@ -245,40 +246,59 @@ func (b *BackendOptions) UseSSL(v bool) *BackendOptions {
 
 // SSLMinVersion sets the minimum allowed TLS version on SSL connections to this backend.
 func (b *BackendOptions) SSLMinVersion(min TLSVersion) *BackendOptions {
+	b.abiOpts.UseSSL(true)
 	b.abiOpts.SSLMinVersion(fastly.TLSVersion(min))
 	return b
 }
 
 // SSLMaxVersion sets the maximum allowed TLS version on SSL connections to this backend.
```

Comment:
```suggestion
// SSLMaxVersion sets the maximum allowed TLS version on SSL connections to this backend.
// Setting this will enable SSL for the connection as a side effect.
```